test/scripts/e2e-kind.sh

```diff
@@ -46,6 +46,10 @@ ClusterDns \[Feature:Example\]
 should set default value on new IngressClass
 # RACE CONDITION IN TEST, SEE https://github.com/kubernetes/kubernetes/pull/90254
 should prevent Ingress creation if more than 1 IngressClass marked as default
+
+# shard-n Tests
+#  See: https://github.com/ovn-org/ovn-kubernetes/issues/1516
+Network.+should resolve connrection reset issue
```

Comment:
anyway, the name is wrong

https://github.com/kubernetes/kubernetes/blob/4de511a4cfdb8e49e804be18245911bf3270fb2f/test/e2e/network/kube_proxy.go#L239

```
ginkgo.It("should resolve connection reset issue #74839 [Slow]
```

Reply:
yeah the name was updated in newer k8s E2E tests (in the older stuff it is still mispelled) we could cover this by conn.* reset issue

Reply:
I didn't notice that the name was changed later, in 1.18 it is like that, don't need to change it